test/TestUtils.py

```diff
@@ -22,16 +22,14 @@
 
 import logging
 import os
-from pathlib import Path
 import subprocess
 import sys
+from pathlib import Path
 
 import pytest
-
-import ansiblelint.utils as utils
-from ansiblelint.file_utils import normpath
+from ansiblelint import cli, utils
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.